apps/avatax/.env.example

```diff
@@ -2,6 +2,7 @@ SECRET_KEY=
 
 # Additional variables for the app - not needed to start the app
 # APL=file # tells app where to keep its credentials
+# APL_FILE_PATH= # path to the file with credentials (if APL is set to file)
```

Comment:
```suggestion
# path to the file with credentials (if APL is set to file).
#  Example: auth/auth.json
# If not set, default will be used
FiLE_APL_FILE_PATH=
```

1. Lets prefix with `FILE_APL` because we dont want to confuse  this env with other APL (not related)
2. Lets extend descriptions to be self documenting. e.g. I'm not sure if path should start with relative path, ot absolute?